src/fsharp/TypeChecker.fs

```diff
@@ -7684,14 +7684,22 @@ and TcComputationExpression cenv env overallTy mWhole (interpExpr: Expr) builder
     let customOperationMethodsIndexedByKeyword = 
         customOperationMethods
         |> Seq.groupBy (fun (nm, _, _, _, _, _, _, _, _) -> nm)
-        |> Seq.map (fun (nm, g) -> (nm, Seq.toList g))
+        |> Seq.map (fun (nm, group) ->
+            (nm,
```

Comment:
Call to `g.langVersion.SupportsFeature NameOfFeature`

Reply:
Do I just add that line in between 7687 and 7688? What NameOfFeature should I use? “FS-1056”?

Reply:
Ah, I see you answered that in your comment below.

Reply:
In this case I think `OverloadsForCustomOperations` would work

Reply:
> Almost there ...

Reply:
> It's away!